src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -34,6 +35,8 @@
 
 public class Tags
 {
+    public static void init () {}; //Dummy init, create the tags so that they are added to their collections.
```

Comment:
This does absolutely nothing. Congrats you run the static tantalizer of the outer class. It doesn't do anything to the inner classes.